lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/LionWebLanguageConverter.kt

```diff
@@ -60,6 +64,17 @@ class LionWebLanguageConverter {
         registerMapping(PlaceholderElement::class, StarLasuLWLanguage.PlaceholderElement)
         registerMapping(Statement::class, StarLasuLWLanguage.Statement)
         registerMapping(TypeAnnotation::class, StarLasuLWLanguage.TypeAnnotation)
+
+        registerMapping(Issue::class, StarLasuLWLanguage.Issue)
+        classesAndEnumerations.associate(
+            IssueSeverity::class,
+            (StarLasuLWLanguage.Issue.getFeatureByName("severity") as Property).type as Enumeration
```

Comment:
I would consider either creating constants for these features, or alternatively to use Issue::severity.name and Issue::type.name, as a way to ensure that if we change the name, this value is updated